File: launchdarkly/typescript-node/src/GetDestinationsExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.DataExportDestinationsApi();
apiCaller.setApiKey(api.DataExportDestinationsApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getDestinations().then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling DataExportDestinations#getDestinations:");
  console.log(error.body);
});

File: launchdarkly/ruby/src/GetDestinationsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::DataExportDestinationsApi.new.get_destinations

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling DataExportDestinationsApi#get_destinations: #{e}"
end
